app/assets/javascripts/discourse/app/components/user-menu/profile-tab-content.hbs

```diff
@@ -120,6 +120,19 @@
     </li>
   {{/if}}
 
+  {{#each this.extraItems as |item|}}
+    <li class={{(or item.class item.className)}}>
```

Comment:
Out of curiosity, why do we need to support `class` and `className` here instead of settling for one?

Reply:
I think `class` is a better name.. but to keep the legacy version of this API working, it's `className`. We can drop support for `class`, but I like it more.

Reply:
I think we should just stick with `className` for now. It might be less ideal readability wise but we at least introduce less confusion or options when using the API.